spec/fixtures/shared_dict.lua

```diff
@@ -0,0 +1,33 @@
+--- resty options executed by bin/busted
+return
```

Comment:
It may have better implementation as we just did:
```
local dicts = {
 "kong 5m",
  ...
}

for i, v in ipairs(dicts) do
  dicts[i] = "--shdict '" .. v .. "'"
end

return table.concat(dicts, " ") 
```

Reply:
done!